modules/rest-api/pages/rest-reader-writer-thread-config.adoc

```diff
@@ -48,7 +47,8 @@ Note that:
 
 * The _AuxIO_ thread pool is used to run _auxiliary I/O_ tasks -- for example, the _access log_ task.
 
-In all cases, the value of `int` must be an integer between `4` and `64`, inclusive.
+In all cases, the value of `int` should be an integer between `0` and `64`, inclusive.
```

Comment:
Apologies, I forgot that we don't allow the value to be set to 0 as default though the ns_server interface. Only the value "default" maybe be used to tell couchbase to calculate the number of threads itself.

Reply:
Ok I've modified.